alws/release_planner.py

```diff
@@ -215,10 +215,10 @@ async def prepare_data_for_executing_async_tasks(
             for repo in self.base_platform.repos:
                 pulp_repo_info = pulp_repos[repo.pulp_href]
                 self.repo_data_by_href[repo.pulp_href] = (repo.id, repo.arch)
-                is_debug = bool(re.search(r'debug(info|source|)',
-                                          pulp_repo_info['name']))
+                repo_is_debug = bool(re.search(r'debug(info|source|)',
```

Comment:
Please, remove last `|`, since `name-debug` package is not a debug package.

Reply:
we looking here for debug repos, not packages, but i'll fix it anyway

Reply:
Sorry, was confused by previous change, than last `|` should stay